package.json

```diff
@@ -1828,30 +1829,15 @@
                 ]
             },
             {
-                "id": "jupyter-notebook-renderer",
-                "entrypoint": "./out/client_renderer/renderers.js",
-                "displayName": "Jupyter Notebook Renderer",
-                "requiresMessaging": "optional",
+                "id": "jupyter-notebook-image-renderer",
+                "entrypoint": "./out/datascience-ui/imageRenderer/imageRenderer.js",
+                "displayName": "Jupyter Image Renderer",
+                "requiresMessaging": "always",
```

Comment:
Jupyter extension ships with rendere for images so we can have save as / open plotviewer icons